.busted

```diff
@@ -12,7 +12,7 @@
 -- See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
 --
 -- Authors:     Alex Dean
--- Copyright:   Copyright (c) 2013 Snowplow Analytics Ltd
+-- Copyright:   Copyright (c) 2022 Snowplow Analytics Ltd
```

Comment:
I think we can remove this bottom part of the header, have done so in other recent trackers (at least the ones I checked). I would remove the lines 13 to 16. But if there is some reason for keeping them that I overlooked, feel free to do so.